README.md

```diff
@@ -79,7 +79,7 @@ Confirm that the platform is up and running by browsing to http://localhost:8080
 3. To build JVM artifacts (the default, includes all projects), from the sbt prompt:
 
 ```shell
-> assembly
+> clean assembly
```

Comment:
This is fine as just `clean assembly` because an early step has you start up `sbt` but I fixed the one on 108. Good catch!